web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -15,7 +15,6 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
     PENDING_EVENT_ATTR: 'PENDING_EVENT',
 
     constructor: function(cfg) {
-
```

Comment:
@danfruehauf 'developer discretion' means that you can choose how you want to write your own code (or code that you update). It *doesn't* mean you get to reformat every file you touch. What a huge waste of time that would be for everyone.

I mentioned this specific issue when we spoke about blank lines in code:
> The thing with developer discretion is we don't want one developer adding blank lines, and another removing them, and then someone else adding them back in.

If you feel strongly about this *please* raise it again in internal discussions and we can come to an agreement (I think the discussion will go in favour of no blank lines so you don't need to worry). Until it is sorted please add these blank lines back in for the purposes of this PR.